Easing.swift

```diff
@@ -548,19 +548,19 @@ private func elasticEaseOut <T: FloatingPointMath> (_ x: T) -> T {
 ///
 /// Modelled after piecewise exponentially-damped sine wave:
 ///
-/// y = 1/2 * sin(13pi/4*x) * pow(2, 10 * ((2*x) - 1))          in  [0,0.5[
+/// y = 1/2 * sin((13pi/2) * 2*x) * pow(2, 10 * ((2*x) - 1))          in  [0,0.5[
```

Comment:
Can you please vertically align the `in  [0,0.5[` with the  `in  [0.5, 1]` of the next line. Thanks!